source/Playnite/Database/GameDatabase.cs

```diff
@@ -712,12 +711,13 @@ public Game ImportGame(GameMetadata metadata)
         public List<Game> ImportGames(LibraryPlugin library, bool forcePlayTimeSync)
         {
             var addedGames = new List<Game>();
-            foreach (var newGame in library.GetGames())
+            var libraryGames = library.GetGames().ToList();

```

Comment:
Why conversion to list? Isn't that unnecessary here?

Reply:
I am using it to [prevent multiple enumeration](https://www.jetbrains.com/help/resharper/2019.1/PossibleMultipleEnumeration.html) as the library games IEnumerable is now used in the foreach and the isInstalled check.